packages/core/src/common/utils/keyboardUtils.ts

```diff
@@ -22,24 +22,25 @@ export function isKeyboardClick(event: React.KeyboardEvent<HTMLElement>) {
     return event.key === "Enter" || event.key === " ";
 }
 
+const ARROW_KEYS = ["ArrowUp" as const, "ArrowDown" as const, "ArrowLeft" as const, "ArrowRight" as const];
```

Comment:
Could we write this as:
```suggestion
const ARROW_KEYS = ["ArrowUp", "ArrowDown", "ArrowLeft", "ArrowRight"] as const;
```

Reply:
Yes my b, good catch